src/components/common/Extensions/UserHub/partials/BalanceTab/partials/StreamsInfoRow/StreamsInfoRow.tsx

```diff
@@ -0,0 +1,186 @@
+import LoadingSkeleton from '~common/LoadingSkeleton/LoadingSkeleton.tsx';
+import { currencySymbolMap } from '~constants/currency.ts';
+import { useAppContext } from '~context/AppContext/AppContext.ts';
+import { useColonyContext } from '~context/ColonyContext/ColonyContext.ts';
+import { useCurrencyContext } from '~context/CurrencyContext/CurrencyContext.ts';
+import { useGetStreamingPaymentsQuery } from '~gql';
+import useCurrentBlockTime from '~hooks/useCurrentBlockTime.ts';
+import Numeral from '~shared/Numeral/Numeral.tsx';
+import { notNull } from '~utils/arrays/index.ts';
+import { formatText } from '~utils/intl.ts';
+// import Button from '~v5/shared/Button/Button.tsx';
+// import NotificationBanner from '~v5/shared/NotificationBanner/NotificationBanner.tsx';
+
+import AvailableToClaimCounter from '../AvailableToClaimCounter/AvailableToClaimCounter.tsx';
+
+import { type StreamingPaymentItems } from './types.ts';
+import { calculateTotalsFromStreams } from './utils.ts';
+
+const displayName =
+  'common.Extensions.UserHub.partials.BalanceTab.partials.StreamsInfoRow';
+
+const MSG = defineMessages({
+  title: {
+    id: `${displayName}.title`,
+    defaultMessage: 'Streams',
+  },
+  totalClaimed: {
+    id: `${displayName}.totalClaimed`,
+    defaultMessage: 'Total claimed',
+  },
+  availableToClaim: {
+    id: `${displayName}.availableToClaim`,
+    defaultMessage: 'Available to claim',
+  },
+  claimAllStreams: {
+    id: `${displayName}.claimAllStreams`,
+    defaultMessage: 'Claim all streams',
+  },
+  viewStreams: {
+    id: `${displayName}.viewStreams`,
+    defaultMessage: 'View streams',
+  },
+  insufficientFundsErrorMessage: {
+    id: `${displayName}.insufficientFundsErrorMessage`,
+    defaultMessage: 'Insufficient funds in teams to claim all funds.',
+  },
+});
+
+const StreamsInfoRow: FC = () => {
+  const { user } = useAppContext();
+  const { walletAddress } = user ?? {};
+  const { currentBlockTime: blockTime } = useCurrentBlockTime();
+
+  const { data, loading } = useGetStreamingPaymentsQuery({
+    variables: {
+      recipientAddress: walletAddress ?? '',
+    },
+    fetchPolicy: 'network-only',
+  });
+
+  const streamingPayments = useMemo(
+    () => data?.listStreamingPayments?.items?.filter(notNull) || [],
+    [data?.listStreamingPayments?.items],
+  );
+
+  const { currency } = useCurrencyContext();
+
+  const { colony } = useColonyContext();
+
+  const [totalFunds, setTotalFunds] = useState<{
+    totalAvailable: number;
+    totalClaimed: number;
+  }>({
+    totalAvailable: 0,
+    totalClaimed: 0,
+  });
+
+  const getTotalFunds = useCallback(
+    async (items: StreamingPaymentItems) => {
+      const { totalAvailable, totalClaimed } = await calculateTotalsFromStreams(
+        {
+          streamingPayments: items,
+          currentTimestamp: Math.floor(blockTime ?? Date.now() / 1000),
+          currency,
+          colony,
+        },
+      );
```

Comment:
I'm not sure this works as it assumes all streaming payments are from the current colony. It seems like the `useGetStreamingPaymentsQuery` gets all the streaming payments for that user from every colony, so we should instead group the streaming payments by colony before passing them to this function - or something like that, there may be a better way to do it.